src/app/ledger/main.c

```diff
@@ -423,6 +428,17 @@ fd_ledger_main_setup( fd_ledger_args_t * args ) {
   fd_bpf_scan_and_create_bpf_program_cache_entry( args->slot_ctx, args->slot_ctx->funk_txn, 1 );
   fd_funk_end_write( funk );
 
+  /* Allocate memory for the account scratch space */
+
+  args->spad_cnt = fd_tpool_worker_cnt( args->tpool );
+  for( ulong i=0UL; i<args->spad_cnt; i++ ) {
+    ulong       total_mem_sz = fd_ulong_align_up( 128UL * FD_ACC_SZ_MAX, FD_SPAD_ALIGN );
+    uchar *     mem          = fd_wksp_alloc_laddr( args->wksp, FD_SPAD_ALIGN, total_mem_sz, 999UL );
```

Comment:
can we add a comment explaining that this isn't thread-local memory, and probably should be in the future